### ISSUE TITLE
Server-Specific

### ISSUE THREAD

**Issue by Xariyl** [Xariyl/FishyExperiment812390#8]
Catch-all for any server-specific notes or issues.

**Comment by Xariyl** [comment]
Latest version of COFH-Core needed Forge 1448; server was running 1395.

Updated 7/12 with latest files from Fishy Dropbox and latest recommended Forge (1448).

**Comment by Xariyl** [comment]
On Tuesday 28th July, Nodecraft is doing some hardware updates to the server and I'll be getting a new IP address. I'll post it as soon as I have the new address.

---

Update: Server migration is complete. New address is: 

23.235.250.99:25565

**Comment by Xariyl** [comment]
modpack v1.0 has been uploaded to the server.

**Comment by odenthefish** [comment]
Odd.... While playing I lost half my inventory while in the mining world.... 
Take frequent screenshots of your inventory and tool stats so if you lose anything you can get it back.
I will have to look into it later.

**Comment by Xariyl** [comment]
Just on the server? Didn't happen in a test world? I'll be back in town next week and I'll do some more testing then. Odd.

**Comment by odenthefish** [comment]
I haven't been able to recreate it ether. I was morphing into a bat at the time.

**Comment by Xariyl** [comment]
I've had weird issues when crossing dimensions while morphing also. One time when crossing to the Nether, I was stuck at 3 hearts even when in human form, was transferring from a bat as I crossed. Haven't had it affect my inventory though, that is weird for me. May be something with the Mining world? I've not used it before.

**Comment by odenthefish** [comment]
Yeah I'm wondering if it's an incompatibility between  mining world and morph.

**Comment by odenthefish** [comment]
Could i get the logs for the server from the 12th at 3:20 to 4:00? I might be able to find out what happened to my inventory and see if it was due to a bad connection or a mod interaction.

**Comment by Xariyl** [comment]
[Server log on Dropbox](https://www.dropbox.com/s/1aej9qz3e5fu0mu/server_2015-08-12-1.log?dl=0)

**Comment by odenthefish** [comment]
Thanks, I haven't been able to test much but I will have a look at the log and see if I can find anything that might help.
It could have been a one time thing like a client server lag issue, I wasn't able to recreate it on the server the little I did test it.
Also yesterday I saw lightning strike about 20Ft from me ò_ø

**Comment by odenthefish** [comment]
I looked through it and didn't see anything, it could have been a client issue.
I think it might have had something to do with my internet.
Until I can reproduce it I'm marking this a one time event, possibly due to internet.
I'll make sure all inventory related mods are updated, I think that inventory tweaks might still be an old DEV version...
EDIT: LOL it is nearly 10 months old!

**Comment by odenthefish** [comment]
I had another event of it happen, I was in my base about to go dig up some snow and half my inventory disappeared and got messed up, https://www.dropbox.com/sh/spy4jscdttom15k/AAAel_-NrXJYi1S4ZkDE69i2a?dl=0 
Have you had any problems with your inventory?
I'm going to try removing a couple client side mods such as MAtmos ( :( ) to see if that fixes it, although I would think that a client side mod would have no impact on a server kept inventory
I will give myself back my inventory later.... when I'm not so frustrated.

**Comment by Xariyl** [comment]
I've had a couple of issues but they were clearly synchronization related; I'd go gather gear from chests, chests wouldn't do their close-animation, and a few seconds later I end up back at where I logged in with my inventory empty again.

There is a bug with Inventory Tweaks also, where when it auto-replenishes a stack in your hotbar the items may appear on  your cursor or seem to go "invisible" in your inventory. Turning off auto-refill helps mitigate that.

My first guess is there is a client/server sync issue on your end. I've had stuff seem to be invisible but never actually remove itself from my inventory. This just started happening after the last update? Let's go over the modlist and see what's changed, maybe we can roll back something.

First off: what client-side mods do you have installed?

Edit: here's the changes to the modlist folder: https://github.com/Xariyl/FishyExperiment812390/commit/a51bcafd8e39f98f27ce90364ff772d38422202e

**Comment by odenthefish** [comment]
Yes it just started, and as far as I can tell it only happens on servers, I have logged a few hours in a world I have that I can destroy (a downloaded city map) and it goes without a hitch, but I mostly die and blow stuff up so I don't know if it occurs and I just die too often to notice.
Here is my client side mods: Lite loader mods:
Link_Info-mc1.7.10-0.3.1 (TBH I don't know if this is even useful...)
mod_Autofish_0.4.4_mc1.7.10 
mod_betterClouds-1.0.1_for_1.7.10 (again not sure of the usefulness)
mod_macros_0.10.12_for_1.7.10 (woot)
mod_MAtmos_29ub4_mc1.7.10 (amazing sound mod, too bad its not supported by the original author anymore....)
mod_noteblockdisplay_1.3.2_mc1.7.10 (if I used noteblocks more I'd love this so much)
PresenceFootsteps_r5c_Unofficial__1.7.10 (adds better stepping sounds than vanilla)
that is it for lite mods I didn't realize how many I had...
that is is for the client mods that aren't in the modpack I believe.

It came to me that this happened within close time between me building stuff with schematica, that was updated last time, and its known that schematica has a few issues with AMD GPUs (gosh I hate this AMD, it might be more powerful than my last GPU but the bad drivers are so annoying)

Traveler's gear was added as well, that heavily deals in inventory.
Thermal smeltery was added... I don't know why but this mod gives me a buggy sort of feel, like its still in alpha/beta.

I'm looking closest at the mods that do stuff with inventory first because I'd think they were the first to cause these kinds of problems.

EDIT: wow after looking at that modslist I'm amazed at how many mods I have that are so old they don't even have a update checker of anykind active, as you can probably guess I picked most of these mods up close to the beginning of the year, or the middle-late of last year, I can't believe how long the 1.8 update cycle has been.

**Comment by Xariyl** [comment]
[Log from last night](https://www.dropbox.com/s/8q2p821x3x6pa29/server_2015-08-17.log?dl=0). Check it out, there's a lot of stuff in there I hadn't seen before.

Confirm that your local mod files are the same versions on the server modlist. There's a ton of block-updates, failed world-saves, and specific InventoryTweaks issues going on, from what I googled they say it's commonly a mismatched file version.

EDIT: Yeah, but the 1.7 updates took almost as long. Really until COFH updates, many other authors won't even bother trying. Those that do, are still actively squashing a ton of bugs, plus Forge keeps updating to fix Mojang bugs, and so on. We probably won't see any major push to 1.8 until after 1.9 comes out, and even then many authors were planning to skip right over it.

**Comment by odenthefish** [comment]
Wow that is full of Debug stuff it appears, I'll have to look at it tomorrow (its actually time for bed for me right now...) 
That is the same log file you got the other log from right? if so then wow there has been a change since the last problem....

I'm pretty sure that my mods are all up to date but I will make sure.

PS I added 3 new rooms to my base :D one of them is a failed project that was meant to have a door that opens when go up to it ( for flying directly into my base instead of having to go to a door) but I can't find a way to detect players in the area I want, also I removed a bunch of snow blocks on the ground next to it so no creepers could sneak onto my roof, that needs terraforming next.

EDIT: wait in the log, the thing where it says [02:32:29] [Server thread/DEBUG] [FML/]: Gathering id map for writing to world save world
[02:33:14] [Server thread/DEBUG] [FML/]: The world ac91416 (world) may have leaked: seen 90 times.
stuff never ends so does that mean it still hasn't saved? perhaps since the update?

**Comment by Xariyl** [comment]
There's that, and there's also this line: "[23:58:07] [Server thread/ERROR] [FML/]: There was a critical exception handling a packet on channel InventoryTweaks." and all the exception-error junk that follows it, occurs several times also.

This log is from 8/17.

**Comment by odenthefish** [comment]
Okay I'm getting on the server at 1:55 AM with the client of experiment 812390 no other mods installed
... later, at 2:25-26 i lost my inventory again, right after giving it all back... 
at 2:30 i lost my hotbar, this is getting more and more common
I'll have to give myself back my inventory later, but I'll keep a backup in a protected resonant strongbox so i can just pickblock on them in creative so i don't have to go through all the rebuilding i always have to

EDIT: i thought i included thermal smeltery but it doesn't exist in the file, eh I might add it later.

**Comment by odenthefish** [comment]
I think that my player data might be corrupted, Today I made a fresh instance of minecraft and logged into the sever and when I went to move some feathers in my inventory I couldn't move them! I have to drag any items in my inventory if I want to move them. 
Also I can't always get the options for NEI to click... 
What would happen if we delete my playerdata? Of course we'd need to do backups of everything.

**Comment by odenthefish** [comment]
It occurs to me that this only started happening to me after I went to the mining world, have you been? if not don't go unless you make a complete server backup, I'm thinking that its the mining world causing this.

**Comment by Xariyl** [comment]
I have stepped in and looked around for a little bit, I went through both of the portals near your base. I haven't had any issues yet.

Creating a full backup now. I won't log in tonight, so if there's anything you want to test go for it, and we can always restore the back up later if needed.

Also finally setting up an auto-backup and restart to run every week.

**Comment by Xariyl** [comment]
I don't know if it makes a difference, but what version of Forge are you running? I have forge-10.13.4.1448 running on the server and on my client.

**Comment by odenthefish** [comment]
Forge 10.13.4.1452
I thought about it being forge but I didn't think it'd cause any adverse affects considering most servers don't share their forge version, I will try updating it.
Maybe they did something silly in a forge update.
EDIT: my fresh instance was using your version, I'll try it anyway.

**Comment by odenthefish** [comment]
I updated my old instance and so far I haven't had any issues, maybe it was all fixed when i updated forge.
Did you reboot the server? That could have been what fixed it.

**Comment by Xariyl** [comment]
I did reboot a couple of times before but you said you were still having issues. So I'm not sure that _that_ alone would have fixed it. I wonder if some newer Forge code may have been playing poopy with one of the existing mods? I don't see anything specific in the changelogs, but that doesn't necessarily mean anything.

**Comment by Xariyl** [comment]
Looks like Nodecraft has forge 1492 finally available. If you still have troubles I will try updating this weekend.

**Comment by Xariyl** [comment]
Saw your note on the BC forum. Before we go fully public I'd like to see about updating to a later Forge version. I'm wondering if that may be the cause behind some of the issues. I'll see about setting it up when I get home from work today, so should be ready by the time you sign on.

**Comment by odenthefish** [comment]
Ok, I'm going to see about using ftb for it, it should be pretty soon.

**Comment by odenthefish** [comment]
You can invite a few people to the server, it'll speed up testing a bit.
Also what forge version is the server using?

**Comment by Xariyl** [comment]
I just installed 10.13.4.1492. No inventory issues so far.

**Comment by odenthefish** [comment]
I built a disposable turret cheaty thing if you want to see one, instead of the high cost of the laser turret or high gunpower cost of other turret types it uses the cheap disposable item turrets coupled with a igneous extruder (I hate that word) to make cheap turrets you can plop down in large numbers.
So far I haven't had any problems with my inventory.

**Comment by odenthefish** [comment]
Okay.... Don't put power armor on.. at least on the server, I did and I keep getting kicked because of it..
I don't know what to do to fix it ether, maybe updating will do the trick but I can't play due to it.
I knew that was going to happen...
I even tried spamming /kill but I can't log in fast enough to do it, even with macros keybind mod

**Comment by odenthefish** [comment]
Perhaps you could set up a command block with /kill Oden_The_Fish  on a quick timer once you get on.
If I could get that helmet off it'd all be good...

**Comment by Xariyl** [comment]
I updated using your latest pack. I haven't used MPS before so I'm not sure I really tested everything as thoroughly as I maybe should have, but no issues found on single player. I'm uploading to the server now. If you still have issues logging in let me know, I'll remove the mod completely and let the items delete themselves.

Server live now as of 9/5 @ 5:30 AM US-Central (UTC-6).

**Comment by odenthefish** [comment]
No it is my fault, I should have tested on my local server because it has very high quality rendering and I believe that there are issues with rendering of things like that in MP.

**Comment by Xariyl** [comment]
Wow... MPS is spamming the console like mad, lol.

[03:40:27] [Server thread/WARN]  [MachineMuse]: WARNING: unlocalizedName is deprecated; please use registryName or itemStackName instead!

**Comment by Xariyl** [comment]
https://github.com/MachineMuse/MachineMusePowersuits/issues/607

The MPS spam is apparently fixed in the latest version of Numina, either that or we have an outdated recipes file (are you using a custom one?). Mind throwing the latest Numina in for the next pack update?

**Comment by odenthefish** [comment]
I do have an older version of Numinia, the other versions were marked BETA so I thought i would wait.
And I do have the auto generated config for recipes uploaded to the pack because I didn't want any recipe issues if we added a mod it has recipe hooks to.
I will update Numinia and probably just delete the config because chances are we won't add anymore mods it has recipes with.

I believe that everything is in order for a public release, I just want to double check the open permissions on FTB and go back over the modslist one more time, because the Dev of new Stefinus 3D guns mod seems to be dead.. and it has quite a few bugs with morph so it might get poofed.
I really like the ability to autocraft the ammo and such instead of the flan's mod way....

I had all the time in the world, then I started working on experiment 812390 and I lost all of it somehow...

I have to allow time to play! Remember all work and no play makes jack a Pahimar :P

**Comment by odenthefish** [comment]
If you _really_ want to we can reset the world but I'd like a copy so I can remember some of the setups I have done.
Most of my time was spent doing nothing :P not really much actual work :)

**Comment by Xariyl** [comment]
No, at this time I don't think it's necessary, just wanted your opinion on it. I restart single player worlds so often, so not really sure how anyone else may feel about it. 

Often when I "finalize" a pack I'll create a new world to make sure of how it plays from the beginning. But other than cleaning up configs for the world gen, the pack hasn't changed so much that it would affect any existing world. So we should be okay.

**Comment by odenthefish** [comment]
That's a nice base you have there.... It'd be a shame if someone put some lights up....

**Comment by Xariyl** [comment]
## My base came down with a case of Blue Measles!

hm, when I place them the center is black instead of blue, though they still seem to emit light. Is there a way to change the color when placing?

**Comment by odenthefish** [comment]
Yeah you have to change the color in the tinker table, you can use mine to mess with it if you need to.
I put it in my LP system so you can request it from any connected request pipes or tables :)

**Comment by Xariyl** [comment]
Try again, Fish. I had the TimeScaler mod installed, thought it was server-side only, but it looked like it was rejecting you. I've removed it and restarted the server so you should be able to get in.

**Comment by odenthefish** [comment]
:P I was just about to ask about where you found time scaler, I'm looking everywhere but can't find anything but GTA V mods and some Russian mod...

**Comment by Xariyl** [comment]
I actually just google searched "minecraft TimeScaler mod". It's on the MC-Forums, not on any of the  distribution sites.

[Here it is](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1291771-timescaler).

**Comment by odenthefish** [comment]
Ah, Chances are I'll add it next update.

**Comment by odenthefish** [comment]
Ok, I have removed a few of my chunk loaders that aren't entirely needed 1 at Experiment slime ( that experiment was silly :P) 2 spot loaders at my Oil pump (because the only reason I was going to use oil is because of Pneumatic craft and that seems bugged at the moment) and 1 at the ender farm ( I connected my LP system to the XP so I could request XP at my base) 
And I'll probably change the nether lava pump to a creative tank in the overworld as it is pretty much cheating in lava anyway and that way it won't kill the server.
EDIT: I can already see an improvement, though now the server is showing low TPS in the overworld
https://www.dropbox.com/s/yq8sfpnpehg71k4/2015-10-10_23.59.22.png?dl=0 It didn't seem to start until I went over to Experiment slime, 2 islands not far from your base, So I blame your base! :P or do I blame you.... I can't make up my mind! :P probably one of your minions started a big stink about unionizing.

**Comment by Xariyl** [comment]
I do have several robots going. I'll start shutting them down.

I only have 3 admin-anchors (RC's chunk loaders) going at the moment. 1 at spawn (shouldn't be necessary, but just in case); 1 at the nether entrance (which we should move to the pump), and 1 at the End entrance. Just to keep the worlds loaded for faster switching. The issues seem to have compounded more recently, though, so I'm leaning towards blaming the robots.

**Comment by odenthefish** [comment]
Yea, I'll probably have an update up to the pack (and with it buildcraft) and maybe that'll help.
Silly unionizing robots.....

**Comment by odenthefish** [comment]
Server has seemed quiet lately.... What did you do with Xariyl Xarbot?!? :P

**Comment by Xariyl** [comment]
I've been on a little bit! But yeah, not too much. My wife just got back into town on Friday night, and I've been a little... distracted.

Plus I've been working undecided about what to work on next. I've got a few projects in mind (and a few in various states of progress) but haven't committed to anything yet.

But I'm still around! Xarbot hasn't taken over and replaced me with an evil clone conditioned by the CIA to quietly work dissent and disruption throughout Minecraftia. And the mailboxes are not being used to smuggle BC bots preprogrammed to deliver all your goods to my tower. Nothing like that at all.

**Comment by odenthefish** [comment]
Oh um uh...
:P
I've been wondering what to work on as well, I've been working on pneumatic craft to get a drone up and running but haven't had much time to play ether, but I'm thinking about trying robots for my item pickups around my base again, the buildcraft update might have fixed some buggy robotic behavior, or maybe it was Xarbot 2000.

**Comment by Xariyl** [comment]
Did you install Xarbot Service Pack 2? It was developed by Maxisoft and everyone says it's the best update they've ever done.

Pneumatic craft.... I'm not sold on it. The failure with the refinery really soured my interest in it, and I don't know that it offers anything that we don't have covered by other mods.

**Comment by odenthefish** [comment]
Yea, the refinery really kind of killed my interest in it as well, last time I played with PC I had a lot of fun but this time it just seems really, I don't know slow? The refinery was a good thing to add so that it isn't required to mess with those silly seeds, but I just haven't liked the refinery, even when it works.

I don't think it really adds anything that other mods don't but the drone and pneumatic helmet are cool, I can hack creepers and make them explode on command with a pneumatic helmet :P

Oh and I tried the update by Maxisoft and it just isn't as optimized as service pack 1, maybe in the future I can have a Xarbot that doesn't make my robots cry.

**Comment by odenthefish** [comment]
I'm going to be out of town for 1 or 2 or 3 or 4 days so I probably won't be online for a bit, though I haven't had much time to play otherwise, what has it been a week since I logged on?

**Comment by Xariyl** [comment]
Likewise I haven't been online too much. I've been stalling until I can get more than 5 consecutive minutes to play. But oh well, such is life.

Have a safe trip, have fun !

**Comment by odenthefish** [comment]
Hello! Have you had any time to play? I haven't had much time and most of the time I do have the internet is slower than a dead snail.
I have been thinking about sailing into the wild blue yonder and making a decorated base to farm bees.(A base with a bunch of decoration and an area for bee farms) 
I've been in the mood for some old school forestry, thermal expansion and buildcraft :P

**Comment by Xariyl** [comment]
Welcome back!
No, I haven't been on too much. If I can't get at least a couple of hours then I generally haven't gone online. Just not worth it, barely getting something started and have to keep stopping.

In the meantime I've been messing around with some 1.8 mods in single player. Forge for 1.8 has a ridiculous amount of bugs. The BC Alpha is mostly functional too (I haven't tested robots yet but pipes and most machines work). If you can work with missing textures I've found a lot of mods are actually playable. 

Honestly I think when MC1.9 comes out things will be much better, more stable. Plus the changes to the End look cool.

Anyway. Not this weekend but hopefully next week I'll get some time to play online again.

**Comment by odenthefish** [comment]
Yea, I've kind of been ready for another update, we've been stuck in 1.7 for what? 2 years now? 
Though the mods took about 6 months to update to a usable state.

Wow I can't believe how long I've played, it just barely seems like I started playing minecraft, I have been playing PC since sometime late October, right after 1.7.2 dropped and I played Xbox 360 for about 3 months before. (Once you've played pc you can never go back to Xbox :P )
Off topic....

Hope to see you online soon!

**Comment by odenthefish** [comment]
Ok, I just logged like 6 hours on the server (Yikes, I know why everyone says I play on my computer constantly..) 

And before that 2 or so hours constructing a massive actually simi good looking ship in creative.

Then I built it in survival, mostly, it is still missing berths, (beds that allow you to sleep on ships woo!) some carpet in the main bedroom (AKA the one bed above deck) and most of the front sail.
All these are wool items, and I really need a wool farm.

I also built a docking connector that extends a bridge with the pull of a lever.

![2015-11-27_06 51 52](https://cloud.githubusercontent.com/assets/9970882/11441625/aacb0828-94d3-11e5-9601-4ef5c6e399e7.png)
![2015-11-27_06 51 37](https://cloud.githubusercontent.com/assets/9970882/11441627/af24b392-94d3-11e5-9b4c-7db72720b614.png)

If you want to check it out next time your online and I'm not this is the location: X 887 Z 1490 Y66 it was really hard getting that ship there :P

Plus I finally upgraded to enderium... my cleaver with full quartz and a ball of moss on it does 49 damage in one swing... And the enderium bolts do like 90 DMG including fire damage.. I'd use them but they are so heavy they fail to fly 15 or so blocks heh.

**Comment by Xariyl** [comment]
Very nice! That looks very cool, I like it.

While I don't have a true sheep "farm" per se, you can use my elementium shears to help gather wool quickly. Get a mana tablet/ring, along with the magnet ring; stand near the sheep pen, then right-click-and-hold the shears like a bow. All the wool will start popping off the sheep. I can usually get a little more than a stack of wool from my pen.

That ship does look very cool. Nice job!

**Comment by odenthefish** [comment]
I really like seeing all the sheepys loose their hair like that :P 

I had stolen your sheepys hair last night and left a few diamonds for it (I didn't take any from you LP system though, just the sheep) and it still wasn't enough :o I had to cheat in a few stacks of wool to get the sail done because I really wanted to move it from the weird place I put it.... :P

But gathering wool is pretty much waiting around because it is an infinite resource, and I really wanted to get everything into place last night so I could build the dock.

I'll sacrifice some diamonds to the lava for it later :P 

I don't know what I did to that ship to make it cost so much in wool... everything else is cheap...

**Comment by odenthefish** [comment]
There seems to be an issue with the railways in the End, not the railways them selves.

I was messing with some stuff so I could put a turret above the doorway ( The endermen keep teleporting out there and not dying) and some water spilled over onto the boarding track for the Exit track, I repaired that and got everything working again, but I pressed shift to get out of the railcart and it placed me in the void, and after lots of testing (Breaking and placing the boarding track) I decided to test the other tracks, the track to and from the spawn in point works fine, or so I thought.

When I was at the final stretch (the part facing north and south instead of east and west) I pressed shift, it placed me in the glass, same with the spawn to exit track, it seems directional, I don't know how long its been going on.

![2015-12-11_08 39 18](https://cloud.githubusercontent.com/assets/9970882/11746714/e84331aa-9fe4-11e5-95c1-99710cf5bd4f.png)
![2015-12-11_08 39 15](https://cloud.githubusercontent.com/assets/9970882/11746726/f550a21a-9fe4-11e5-9238-0dd6919b8ee7.png)
![2015-12-11_08 36 51](https://cloud.githubusercontent.com/assets/9970882/11746738/fe24d136-9fe4-11e5-9f44-3a76e57a9357.png)
![2015-12-11_08 36 28](https://cloud.githubusercontent.com/assets/9970882/11746745/03d91de4-9fe5-11e5-83fd-1aa6778e404f.png)

I don't think I broke all of it with a stream of water, but I don't have a clue what is going on.

**Comment by Xariyl** [comment]
It all looked and seemed to work okay to me. The confusion may be that you were shifting out to leave the cart early, and I didn't build in any protection for that. It was designed to just take you to the exit and boot you there. I ran it a few times and it all seemed to work fine if you let it take you all the way to the exit. Boarding track worked as it is supposed to, as soon as a player is in, it takes off. The ejector track is there from when I was testing the rail, I'd ride back and forth and it made sure I landed on the walkway.

**Comment by odenthefish** [comment]
Ah, I was just making sure, I had thought I shifted off by accident before and landed correctly, but I was probably just being crazy :P 
I wanted to make sure you knew so if you didn't have flight you wouldn't die heh.

**Comment by Xariyl** [comment]
Finally, the wither-skelly farm is up!
It's on the roof of the nether. The mob-drop will NOT kill the mobs at the base of the structure, because wither-skulls require a player-kill in order to drop. I stood AFK for about 10 minutes while eating, then went to kill them; picked up 14 wither bones and 7 skulls. Last night I got 13 skulls! 

At the base of the structure are three elevators, (2) black and (1) red. 

The Red will take up up to the "ribs" of the structure, outside the spawning chamber. Everything is half-slabbed so no mobs should spawn, it's a fairly safe AFK platform.

The two Black elevators will take you to a "safe" spot inside the spawning room itself, so you can watch them drop down the shaft. "Safe" in quotes, because sometimes the normal skeletons spawn and can shoot you through the corners of the barrier glass, but they usually fall through soon and don't do much damage.

When you come back from AFK, ride the elevator down to the base of the structure and wack at the captured boneheads. (TiC Cleavers with Loot III work awesome!). Not the most efficient machine, I guess, but it works well enough and I'm rather pleased with it.

**Comment by odenthefish** [comment]
Sometime the bestest machine in the most inefficient :P 
I can't wait to hop on and look at it :)

**Comment by Xariyl** [comment]
Got a couple other ideas I want to try too. Might redo the Blaze and a Zombie Pigman spawners (these we can do an auto-killing setup if we want), and I'm very tempted to try out a Ghast catcher of some sort. Time permitted, of course :)

**Comment by odenthefish** [comment]
A Ghast catcher... Sign me up! I want to catch Ghasts and kill them in cruel and complex ways!

**Comment by Xariyl** [comment]
Lol... I had a vision of you walking through my base with a Ghast on a leash floating behind you like a giant helium balloon.

**Comment by odenthefish** [comment]
That'll happen now.

**Comment by odenthefish** [comment]
Note to self: Don't shift click a Forge multipart block with a Lexica Botania.
It crashes :P

**Comment by Xariyl** [comment]
FYI: on Jan 11 the server will be upgraded and moved to a new IP address. I'll let you know when I get the new address.

**Comment by odenthefish** [comment]
Alright! 
I've been wanting to thank you for all you've done, keeping the server running _and_ dealing with all my sillyness :)

**Comment by Xariyl** [comment]
Nodecraft server maintenance completed. New IP address is: 69.175.6.88

**Comment by Xariyl** [comment]
Hey Oden,
Since it seems no one has been playing on the server for quite a while, would you have any objections about me shutting it down? I don't see much sense in continuing to pay for it if no one is using it.

**Comment by odenthefish** [comment]
Yea it's fine with me, I'm sorry I haven't been on and pretty much dissapeared but I've been really busy and on top of that I haven't been really enjoying minecraft, I'm going to try and update the modpack once more and then put it on hold until further notice.
I would like to have a copy of the world though, to remember the world and server in the future.

**Comment by Xariyl** [comment]
No worries, I haven't been on for a few months now, and I've been playing other stuff besides Minecraft too.

I hit the Cancel button a little too late though, so it's available for another month. Here's the world download as it stands now. 

https://www.dropbox.com/s/79k6l1lacrfkaij/Experiment%20812390%20server-world.zip?dl=0
